Sources/RealModule/Angle.swift

```diff
@@ -160,6 +160,12 @@ public extension Angle {
     }
 }
 
+extension Angle: Equatable {
+    public static func == (lhs: Angle<T>, rhs: Angle<T>) -> Bool {
+        lhs.radians.isApproximatelyEqual(to: rhs.radians)
```

Comment:
Implementations of `==` cannot use approximate equality because it does not fulfill the semantic requirements of `==`.